metrics/prometheus.go

```diff
@@ -500,6 +500,11 @@ func (c *PrometheusCollector) Collect(ch chan<- prometheus.Metric) {
 	c.errors.Collect(ch)
 }
 
+const (
+	dockerLabelPrefix = "docker_label_"
+	dockerEnvPrefix   = "docker_env_"
```

Comment:
What about other runtimes (rkt)? You could use the namespace from the ContainerReference.